### PR TITLE
Added RELEASE_PROCESS document

### PR DESCRIPTION
**Pull request by bdunogier**
The TL;DR duplicates what's below, but maybe some details are interesting to keep.
The part at the end about the PHP version needs to be finished.

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Definition
+
+A release is mostly a git tag of http://github.com/wallabag/wallabag, following [semantic versioning](http://semver.org).
+The last release at the time of writing is 2.0.0-alpha.2, from the v2 branch.
+
+### TL;DR
+- create a branch for the release, from `v2`: `git checkout -b release-2.0.0-alpha.3`
+- run composer update to make sure `composer.lock` is up-to-date
```
Comment (j0k3r) [inline RELEASE_PROCESS.md]: Should we do `composer up --no-dev` for the release?

Comment (nicosomb) [inline RELEASE_PROCESS.md]: yes, I think we can do your command, @j0k3r. For next alpha, I just did `composer update`.

**Comment by j0k3r** [comment]
Could you add a note about updating the config.yml file to set the new release ? (https://github.com/wallabag/wallabag/pull/1612/files#diff-bf0e70bcef1a5d5b2f87289220a51108R31)

**Comment by nicosomb** [comment]
@bdunogier need help on this PR?

**Comment by bdunogier** [comment]
> Could you add a note about updating the config.yml file to set the new release ? 

Done.

Removed the part about the release branch, following experience from our recent release. Ahem.
Anyway. About `composer.lock`, you don't want it in the `v2` branch, do you ? The doc needs to be updated if you do.

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## Definition
+
+A release is mostly a git tag of http://github.com/wallabag/wallabag, following [semantic versioning](http://semver.org).
+The last release at the time of writing is 2.0.0-alpha.2, from the v2 branch.
+
+### Steps
+- Update `wallabag.version` is up-to-date in `app/config/config.yml` if necessary
```
Comment (j0k3r) [inline RELEASE_PROCESS.md]: This sentence looks weird to me. Should you remove the "is up-to-date" ?

**Comment by j0k3r** [comment]
> Removed the part about the release branch, following experience from our recent release. Ahem.

Can you elaborate?

**Comment by j0k3r** [comment]
Could you please rebase your branch against the v2?
We've put a Github token to avoid error like `Could not authenticate against github.com`.

**Comment by nicosomb** [comment]
Is this PR good to be merged?

**Pull request merged**